dcap/types/src/quote3.rs

```diff
@@ -19,13 +23,11 @@ use subtle::ConstantTimeEq;
 // ok on any platform using these types.
 const_assert!(mem::size_of::<usize>() >= mem::size_of::<u32>());
 
-// The size of the quote bytes. Not including the authentication or
-// certification data.
-const QUOTE_SIZE: usize =
-    mem::size_of::<sgx_quote3_t>() + mem::size_of::<sgx_ql_ecdsa_sig_data_t>();
+// Size of the Key
+const KEY_SIZE: usize = 64;
 
-// The offset to the authentication data
-const AUTH_DATA_OFFSET: usize = QUOTE_SIZE;
+// Size of a Signature
+const SIGNATURE_SIZE: usize = 64;
```

Comment:
This and the `MIN_SIGNATURE_DATA_SIZE` are pretty similar up for ideas on better names, they are local constants so it probably isn't super critical name wise